### PR TITLE
TextReading pipeline into `main`

### PR DESCRIPTION
**Pull request by enoriega**
All the changes I think were necessary to merge into main have been there for a while. We can incorporate this and start branching off for newer improvements

### CODE REVIEW

File: scripts/grounding/.travis.yml.old [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sudo: required
```
Comment (adarshp) [inline scripts/grounding/.travis.yml.old:1]: Since we are using Drone for CI, can we remove this file?

File: scripts/grounding/assembly.sbt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+assembly / aggregate := false
+assembly / assemblyMergeStrategy := {
+/*
+  // See https://github.com/sbt/sbt-assembly.
```
Comment (adarshp) [inline scripts/grounding/assembly.sbt:4]: Do we need to keep this commented-out code?

File: scripts/grounding/build.sbt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+val scala12 = "2.12.15" // up to 2.12.14
+val scala13 = "2.13.7"  // up to 2.13.7
+val scala3  = "3.1.0"   // up to 3.1.0
+
+ThisBuild / crossScalaVersions := Seq(scala12, scala11, scala13, scala3)
+ThisBuild / scalaVersion := crossScalaVersions.value.head
+
+resolvers ++= Seq(
+  ("Artifactory" at "http://artifactory.cs.arizona.edu:8081/artifactory/sbt-release").withAllowInsecureProtocol(true)
+//  Resolvers.localResolver,  // Reserve for Two Six.
```
Comment (adarshp) [inline scripts/grounding/build.sbt:18]: Do we need to keep this commented0out code?

File: skema/text_reading/notebooks/skema_text_reading_demo.ipynb [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    "\t}\n",
+    "\n",
+    "\treturn papers.get(doi)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 109,
+   "metadata": {},
+   "outputs": [
```
Comment (adarshp) [inline skema/text_reading/notebooks/skema_text_reading_demo.ipynb:106]: I would like to not commit output cells of Jupyter notebooks to the repo. Can you clear the outputs and re-commit the file?

File: skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Examples from CoNLL 2000 shared task: Chunking.
```
Comment (adarshp) [inline skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README:1]: Do we need these folders: 

- `skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs`
- `skema/text_reading/text_reading/webapp/public/brat/static/case-studies`
- `skema/text_reading/text_reading/input/grfn`

Comment (adarshp) [reply skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README:1]: If not, can we get rid of them?

**Review by adarshp — CHANGES_REQUESTED** [review]
Thanks @enoriega and @kwalcock ! A couple of comments:

- There seems to be a duplicate copy of the grounding code in the `scripts/grounding` directory. Is this intentional?
- Do we need to keep the file `skema/text_reading/text_reading/input/SIR/grfn/SIR-Gillespie-SD_GrFN.json`? I had thought we were moving on to the GroMEt format.

File: scripts/grounding/.github/workflows/ci.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+name: grounding CI
```
[the remark this answers is not in the document]
Comment (enoriega) [reply scripts/grounding/.github/workflows/ci.yml:1]: Yes

File: scripts/grounding/.travis.yml.old (hunk 1 above)
Comment (enoriega) [reply scripts/grounding/.travis.yml.old:1]: Yes

File: scripts/grounding/assembly.sbt (hunk 2 above)
Comment (enoriega) [reply scripts/grounding/assembly.sbt:4]: Noe

File: scripts/grounding/build.sbt (hunk 3 above)
Comment (enoriega) [reply scripts/grounding/build.sbt:18]: No

File: .gitignore [hunk 7]
```diff
@@ -37,7 +37,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+# lib/ # The webapp has some files in a lib directory.
```
[the remark this answers is not in the document]
Comment (enoriega) [reply .gitignore]: Yes

File: skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README (hunk 5 above)
Comment (enoriega) [reply skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README:1]: We don't need them, I removed them

File: skema/text_reading/notebooks/skema_text_reading_demo.ipynb (hunk 4 above)
Comment (enoriega) [reply skema/text_reading/notebooks/skema_text_reading_demo.ipynb:106]: Yes

**Comment by enoriega** [comment]
> Thanks @enoriega and @kwalcock ! A couple of comments:
> 
>     * There seems to be a duplicate copy of the grounding code in the `scripts/grounding` directory. Is this intentional?
> 
>     * Do we need to keep the file `skema/text_reading/text_reading/input/SIR/grfn/SIR-Gillespie-SD_GrFN.json`? I had thought we were moving on to the GroMEt format.

@adarshp the duplicate  directory was supposed to be moved, so I cleared it out. The griffin file is a leftover from automates. I removed it too. There may be some lingering automates data files that we didn't catch. I will remove them as they are found

**Comment by adarshp** [comment]
Thanks @enoriega  - looks good to me!

**Review by adarshp — APPROVED** [review]

**Pull request merged**